### DISCUSSION TITLE
Error sharing a pnpm workspace package using nuxt layer

### DISCUSSION THREAD

**Discussion by yugal-compro** [nuxt/nuxt#26008] [Questions]
I have a pnpm monorepo in which I have
- a base layer
- an app app1 extending base
- base layer using 2 packages - 
    - supervillains 
    - good(local pnpm workspace package)

**Issue:** I am able to use supervillains in app1 but not local package (pnpm workspace)

Able to see my package - good in node modules of base
![image](https://github.com/nuxt/nuxt/assets/107843548/0e3c9db7-147a-4cdf-ac60-4906d43b4220)

In app1-
![image](https://github.com/nuxt/nuxt/assets/107843548/b28a878f-5971-4fb2-88e5-53f98801cc62)

Github repo link - https://github.com/yugal-compro/nuxt-monorepo

**Comment by xjccc** [comment]
In your layers, `good` from `workspace`
```
 "dependencies": {
    "supervillains": "^3.1.0",
    "good": "workspace:^"
  }

```

but in your root `package.json`, `good` is not install?
